pallets/validators-manager/src/lib.rs

```diff
@@ -234,7 +233,7 @@ pub mod pallet {
                 self.validators.len()
             );
             for (validator_account_id, eth_public_key) in &self.validators {
-                <ValidatorAccountIds<T>>::append(validator_account_id);
+                assert_ok!(<ValidatorAccountIds<T>>::try_append(validator_account_id));
```

Comment:
```suggestion
                <ValidatorAccountIds<T>>::try_append(validator_account_id).expect("Too many validator accounts in genesis");
```

Reply:
I prefer the assertion as it is more compact.

Reply:
But the assertion doesn't give you a reason for the error. This one is more explicit and provides a meaningful error.